app/routes.js

```diff
@@ -169,7 +170,10 @@ module.exports.bind = function (app) {
   app.post(registerUser.otpVerify, registerController.submitOtpVerify)
   app.get(registerUser.reVerifyPhone, registerController.showReVerifyPhone)
   app.post(registerUser.reVerifyPhone, registerController.submitReVerifyPhone)
-  app.get(registerUser.logUserIn, loginController.loginAfterRegister, userIsAuthorised, rootController.get)
+  app.get(
+    registerUser.logUserIn,
+    passport.authenticate('localStrategyLoginDirectAfterRegistration', { failureRedirect: '/login' }),
```

Comment:
```suggestion
    passport.authenticate('localStrategyLoginDirectAfterRegistration', { failureRedirect: users.logIn }),
```

Reply:
Thanks, just pushed another commit to fix this